rag-localdata/ownData.py

```
@@ -41,6 +54,31 @@
          ],
      }
 )
+# Get the completion as a Python dict
+completion_dict = completion.model_dump()
+
+# Recursively replace 'content' keys with 'long_text'
+def replace_content_key(obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `replace_content_key`

(ANN201)